.travis/stages/verify/find-unused-or-unmatched-tests/find-orphaned-tests

```diff
@@ -56,6 +57,7 @@ if [ "$status" -eq 1 ]; then
   echo " - renaming the test file to end with 'IntegrationTest'"
   echo " - adding an @Integration annotation to the test"
   echo " - adding an @ExtendedUnitTest annotation to the test"
+  echo " - adding an @SuppressWarngins(\"UnmatchedTest\") to the test"
```

Comment:
Looks like you have a typo here.

Reply:
Good catch! Certainly would be useful to have something that is easily copy/pasted.

Fixed in: cb12baa36